docs/support.md

```diff
@@ -0,0 +1,12 @@
+## Currently Windows and WSL2 do not support `yast`.
```

Comment:
yast does not support windows and wsl2.

Reply:
Just to mention, whenever working on docs keep the punctuation and the styles you use consistent. Non-consistent docs show unprofessional behavior. This will significantly decrease your time when working in non-code PRs